backend/slack_commands/commands/status.py

```
@@ -0,0 +1,65 @@
+        try:
+            status = self._client.get_session_status(session_id, command.user_name)
+            meta_resp = self._client.get_session_meta(session_id, command.user_name)
+        except requests.HTTPError as e:
+            if e.response is not None and e.response.status_code == 404:
+                return SlackResponse(text=f":x: Session `{session_id}` not found.")
+            logger.error("Status check failed: %s", e, exc_info=True)
+            return SlackResponse(text=f":x: Failed to get status: {e}")
+        except requests.Timeout:
+            return SlackResponse(text=":hourglass: Multi-agent service timed out.")
+        except Exception as e:
+            logger.error("Status check failed: %s", e, exc_info=True)
```

Ruff (0.15.15)

[warning] Logging `.exception(...)` should be used instead of `.error(..., exc_info=True)`

(G201)